crates/re_data_store/benches/data_store.rs

```diff
@@ -86,14 +95,62 @@ fn insert(c: &mut Criterion) {
     }
 }
 
+fn insert_same_time_point(c: &mut Criterion) {
+    // Benchmark a corner-case where all rows have the same time point, and arrive out-of-order.
+    // See https://github.com/rerun-io/rerun/issues/4415
+
+    for num_rows in [1000, 10_000, 100_000] {
```

Comment:
Did you manage to run it with 100k rows on your machine? It gives me an estimated 1500s compute time :sweat_smile:

Reply:
Oops, I thought I did… I'l lower the number of run to 10, and lower the number of entities to 50k.